Plugins/Flow.Launcher.Plugin.Program/FileChangeWatcher.cs

```diff
@@ -1,58 +0,0 @@
-﻿using System.Collections.Generic;
-using System.IO;
-using System.Threading.Tasks;
-using Flow.Launcher.Infrastructure.Logger;
-using Flow.Launcher.Plugin.Program.Programs;
-
-namespace Flow.Launcher.Plugin.Program
-{
-    //internal static class FileChangeWatcher
```

Comment:
why is this deleted sorry?

Reply:
It's unused. The whole class is commented long ago.